src/app/content/studyGuides/types.ts

```diff
@@ -0,0 +1,9 @@
+import { HighlightsSummary } from '@openstax/highlighter/dist/api';
+
+// Temporary until we expand this state with more data
+// tslint:disable-next-line: no-empty-interface
+export interface StudyGuides extends HighlightsSummary {}
+
+export interface State {
+  studyGuides: StudyGuides | null;
+}
```

Comment:
This makes the state be `content.studyGuides.studyGuides`, it could probably be changed.

Reply:
I've changed it to `summary` for now, but it will be changed later anyway so I think we should not focus on it too much